From 7b91661b411a521c29f2e3d512bd155a99b04ad4 Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Thu, 24 May 2012 16:39:49 -0500
Subject: [PATCH 320/449] ENGR00210831 - EPDC fb unit test: Collision Test
 fixed by adding delay

EPDC Collision Test (#13) was failing because there was not sufficient
delay between the two updates that should be colliding.  Adding a delay
of 400ms results in the collision being consistently reported.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 test/mxc_fb_test/mxc_epdc_fb_test.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/mxc_fb_test/mxc_epdc_fb_test.c b/test/mxc_fb_test/mxc_epdc_fb_test.c
index 72737aa..a9ce2ee 100644
--- a/test/mxc_fb_test/mxc_epdc_fb_test.c
+++ b/test/mxc_fb_test/mxc_epdc_fb_test.c
@@ -1849,6 +1849,8 @@ static int test_dry_run(void)
 	draw_rectangle(fb, 0, 0, 300, 250, 0x8410);
 	update_to_display(0, 0, 300, 250, WAVEFORM_MODE_AUTO, FALSE, 0);
 
+	usleep(400);
+
 	/* Draw overlapping rectangle, use dry-run test to check for collision */
 	draw_rectangle(fb, 250, 200, 300, 250, 0x4104);
 	coll = update_to_display(250, 200, 300, 250, WAVEFORM_MODE_AUTO, FALSE, EPDC_FLAG_TEST_COLLISION);
-- 
1.8.0

